consensus/src/consensusdb/schema/dag/mod.rs

```diff
@@ -95,3 +121,32 @@ impl ValueCodec<CertifiedNodeSchema> for CertifiedNode {
         Ok(bcs::from_bytes(data)?)
     }
 }
+
+pub const DECOUPLED_PAYLOAD_CF_NAME: ColumnFamilyName = "decoupled_payload";
```

Comment:
why this needs its own CF, isn't it part of the node message?

Reply:
we want to separate Node and Payload separately right? Also, NodeMessage schema is for storing the node before broadcasting, so it serves a different purpose.